src/ace/managers/copper.c

```diff
@@ -57,19 +57,285 @@ void copSwapBuffers(void) {
+    "BLTADAT",
+    NULL,
+    "SPRHDAT",
+    NULL,
+    "DENISEID",
+    "DSKSYNC",
+    "COP1LCH",
+    "COP1LCL",
+    "COP2LCH",
+    "COP2LCL",
+    "COPJMP1",
+    "COPJMP2",
+    "COPINS",
+    "DIWSTRT",
+    "DIWSTOP",
+    "DDFSTRT",
+    "DDFSTOP",
+    "DMACON",
+    "CLXCON",
+    "INTENA",
+    "INTREQ",
+    "ADKCON",
+    "AUD0LCH",
+    "AUD0LCL",
+    "AUD0LEN",
+    "AUD0PER",
+    "AUD0VOL",
+    "AUD0DAT",
+    NULL,
+    NULL,
+    "AUD1LCH",
+    "AUD1LCL",
+    "AUD1LEN",
+    "AUD1PER",
+    "AUD1VOL",
+    "AUD1DAT",
+    NULL,
+    NULL,
+    "AUD2LCH",
+    "AUD2LCL",
+    "AUD2LEN",
+    "AUD2PER",
+    "AUD2VOL",
+    "AUD2DAT",
+    NULL,
+    NULL,
+    "AUD3LCH",
+    "AUD3LCL",
+    "AUD3LEN",
+    "AUD3PER",
+    "AUD3VOL",
+    "AUD3DAT",
+    NULL,
+    NULL,
+    "BPL1PTH",
+    "BPL1PTL",
+    "BPL2PTH",
+    "BPL2PTL",
+    "BPL3PTH",
+    "BPL3PTL",
+    "BPL4PTH",
+    "BPL4PTL",
+    "BPL5PTH",
+    "BPL5PTL",
+    "BPL6PTH",
+    "BPL6PTL",
+    NULL,
+    NULL,
+    NULL,
+    NULL,
+    "BPLCON0",
+    "BPLCON1",
+    "BPLCON2",
+    "BPLCON3",
+    "BPL1MOD",
+    "BPL2MOD",
+    NULL,
+    NULL,
+    "BPL1DAT",
+    "BPL2DAT",
+    "BPL3DAT",
+    "BPL4DAT",
+    "BPL5DAT",
+    "BPL6DAT",
+    NULL,
+    NULL,
+    "SPR0PTH",
+    "SPR0PTL",
+    "SPR1PTH",
+    "SPR1PTL",
+    "SPR2PTH",
+    "SPR2PTL",
+    "SPR3PTH",
+    "SPR3PTL",
+    "SPR4PTH",
+    "SPR4PTL",
+    "SPR5PTH",
+    "SPR5PTL",
+    "SPR6PTH",
+    "SPR6PTL",
+    "SPR7PTH",
+    "SPR7PTL",
+    "SPR0POS",
+    "SPR0CTL",
+    "SPR0DATA",
+    "SPR0DATB",
+    "SPR1POS",
+    "SPR1CTL",
+    "SPR1DATA",
+    "SPR1DATB",
+    "SPR2POS",
+    "SPR2CTL",
+    "SPR2DATA",
+    "SPR2DATB",
+    "SPR3POS",
+    "SPR3CTL",
+    "SPR3DATA",
+    "SPR3DATB",
+    "SPR4POS",
+    "SPR4CTL",
+    "SPR4DATA",
+    "SPR4DATB",
+    "SPR5POS",
+    "SPR5CTL",
+    "SPR5DATA",
+    "SPR5DATB",
+    "SPR6POS",
+    "SPR6CTL",
+    "SPR6DATA",
+    "SPR6DATB",
+    "SPR7POS",
+    "SPR7CTL",
+    "SPR7DATA",
+    "SPR7DATB",
+    "COLOR00",
+    "COLOR01",
+    "COLOR02",
+    "COLOR03",
+    "COLOR04",
+    "COLOR05",
+    "COLOR06",
+    "COLOR07",
+    "COLOR08",
+    "COLOR09",
+    "COLOR10",
+    "COLOR11",
+    "COLOR12",
+    "COLOR13",
+    "COLOR14",
+    "COLOR15",
+    "COLOR16",
+    "COLOR17",
+    "COLOR18",
+    "COLOR19",
+    "COLOR20",
+    "COLOR21",
+    "COLOR22",
+    "COLOR23",
+    "COLOR24",
+    "COLOR25",
+    "COLOR26",
+    "COLOR27",
+    "COLOR28",
+    "COLOR29",
+    "COLOR30",
+    "COLOR31",
+    "HTOTAL",
+    "HSSTOP",
+    "HBSTRT",
+    "HBSTOP",
+    "VTOTAL",
+    "VSSTOP",
+    "VBSTRT",
+    "VBSTOP",
+    NULL,
+    NULL,
+    NULL,
+    NULL,
+    NULL,
+    NULL,
+    "BEAMCON0",
+    "HSSTRT",
+    "VSSTRT",
+    "HCENTER",
+    "DIWHIGH",
+    NULL
+};
+#endif
+
 void copDumpCmd(tCopCmd *pCmd) {
 	if(pCmd->sWait.bfIsWait) {
+		const char* cmd = pCmd->sWait.bfIsSkip ? "SKIP" : "WAIT";
```

Comment:
please use prefix notation. For zero-terminated strings, it's `sz`, so should be `szCmd`